examples/flutter_gallery/pubspec.yaml

```diff
@@ -10,7 +10,7 @@ dependencies:
     path: ../../packages/flutter_sprites
   flutter_markdown:
     path: ../../packages/flutter_markdown
-  flutter_gallery_assets: '0.0.16'
+  flutter_gallery_assets: '0.0.19'
```

Comment:
You may need to update the complex_layout benchmark's reference to flutter_gallery_assets to keep the analyzer happy